[skip gpuci] Feat/cibuildwheel (#11790)
rapidsai/cudf: cpp/CMakeLists.txt

option(BUILD_BENCHMARKS "Configure CMake to build (google & nvbench) benchmarks" OFF)
option(BUILD_SHARED_LIBS "Build cuDF shared libraries" ON)
option(JITIFY_USE_CACHE "Use a file cache for JIT compiled kernels" ON)

> option(CUDF_BUILD_WHEELS "Whether we're building libcudf to go in a wheel for pypi" OFF)

Reviewer: Avoid "we". Also no need to specify PyPI here.
```suggestion
option(CUDF_BUILD_WHEELS "Whether libcudf is being built for a Python wheel" OFF)
```
